mackerel-plugin-mailq/lib/main_test.go

```diff
@@ -267,15 +287,21 @@ func TestFetchMetricsPostfix(t *testing.T) {
 		labelPrefix: "Mailq",
 	}
 
-	{
-		os.Setenv("TEST_MAILQ_COUNT", "42")
+	for _, tc := range []struct {
+		mailqCount int
+	}{
+		{mailqCount: 42},
+		{mailqCount: 0},
+		{mailqCount: 1}, // #1185
+	} {
+		os.Setenv("TEST_MAILQ_COUNT", strconv.Itoa(tc.mailqCount))
```

Comment:
[note] In general, Go 1.17 or higher, it would be better to use *testing.T.Setenv*.

But to keep consistency this use *os.Setenv* here intentionally because all test cases other than that still use *os.Getenv.*